API.md

```diff
@@ -0,0 +1,307 @@
+# API Documentation
+
+Sorteramera is an application made for listing the nearest recycle locations.
+This is the documentation for the Sortera Mera API. All info is obtainable without [authorization](#Authorisation), however updating entries can only be done by admins.
+
+### Terminology
+
+* **Recycle location**
+
+	A recycle location is a recycle center or station. The entry in the 	database contains information about opening hours, city, materials you can recycle etc.
+
+
+* **Change request**
+
+	If the info provided about a recycle location is faulty a change request can be made. The change request will be saved separately and not be applied until approved by an admin.
+
+
+## Get closest recycle locations
+
+To get nearby recycle locations you make a `GET` request containing longitude and latitude of your location.
+
+* **URL**
+
+	`/recycle_locations`
+* **Method**
+
+	`GET`
+* **URL params**
+
+	**Required**
+
+    `longitude=[float]`
+
+    `latitude=[float]`
+
+* **Success**
+
+	**Code:** 200
+
+    **Content:**
+
+    `JSON` array containing all the nearby recycle locations.
+
+* **Example**
+
+	**Command:** `curl '/recycle_locations/longitude=15&latitude=56'`
```

Comment:
The curl command can't use relative paths. You need to have the host and protocol as well.